content/en/funding/hackathon-funding.mdx

```diff
@@ -104,15 +103,15 @@ You can find out how to on our [content page](/work_with_us/content/).
 
 #### Event Details
 
-- The theme of the event, if any
+- The theme of the event \(optional\)
```

Comment:
is it optional though? Most events have a theme or topic. I think , if any suggests a more thorough application